### PR TITLE
Moving over the core Sumologic classes.

### PR DESCRIPTION
**Pull request by bschilt**

### CODE REVIEW

File: src/Audit/SumoLogicEmptyQuery.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Drutiny\Acquia\CS\Audit;
```
Comment (fiasco) [inline src/Audit/SumoLogicEmptyQuery.php]: This needs updating

File: src/Audit/SumoLogicQuery.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drutiny\Acquia\CS\Audit;
```
Comment (fiasco) [inline src/Audit/SumoLogicQuery.php]: Update

File: src/Audit/SumologicTimeSeriesGraph.php [hunk 3]
```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drutiny\Acquia\CS\Audit;
```
Comment (fiasco) [inline src/Audit/SumologicTimeSeriesGraph.php]: Update

**Review by fiasco — CHANGES_REQUESTED** [review]

**Comment by bschilt** [comment]
Great catch, @fiasco! Rookie move. I have updated the namespace.

**Comment by fiasco** [comment]
Thanks Brian!

**Pull request merged**